recipes/repository.rb

```diff
@@ -19,7 +19,22 @@
 
 case node['platform_family']
 when 'fedora', 'rhel'
```

Comment:
Yeah, it's `rhel` right now although there is an open request to change to `fedora`.  Regardless, not its own `platform_family`.